src/github.com/getlantern/fronted/dialer.go

```diff
@@ -183,14 +183,15 @@ func (d *dialer) Close() error {
 }
 
 func (d *dialer) HttpClientUsing(masquerade *Masquerade) *http.Client {
-	enproxyConfig := d.enproxyConfigWith(func(addr string) (net.Conn, error) {
-		return d.dialServerWith(masquerade)
-	})
-
 	return &http.Client{
-		Transport: &http.Transport{
-			Dial: func(network, addr string) (net.Conn, error) {
-				return enproxy.Dial(addr, enproxyConfig)
+		Transport: &DirectDomainTransport{
+			Transport: http.Transport{
```

Comment:
It is what `NewDirectDomainFronter()` doing previously. `DirectDomainTransport` is a simple wrapper of `http.Transport`